TransferHelperApp/src/app/player/add-edit-player/add-edit-player.component.ts

```diff
@@ -7,9 +9,79 @@ import { Component, OnInit } from '@angular/core';
 })
 export class AddEditPlayerComponent implements OnInit {
 
-  constructor() { }
+  playerList$!:Observable<any[]>;
+  positionsList$!:Observable<any[]>;
+
+  constructor(private service:PlayerApiService) { }
+
+  @Input() player:any;
+  id: number = 0;
+  name: string = "";
+  surname: string = "";
+  birthdate: Date = new Date();
+  positionId!: number;
 
   ngOnInit(): void {
+    this.id = this.player.id;
+    this.name = this.player.name;
+    this.surname = this.player.surname;
+    this.birthdate = this.player.birthdate;
+    this.positionId = this.player.positionId;
+    this.playerList$ = this.service.getPlayerList();
+    this.positionsList$ = this.service.getPositionsList();
   }
 
+  addPlayer(){
+    var player = {
+      name:this.name,
+      surname:this.surname,
+      birthdate:this.birthdate,
+      positionId:this.positionId
+    }
+    this.service.addPlayer(player).subscribe(res => {
+      var closeModalBtn = document.getElementById('add-edit-modal-close');
+      if(closeModalBtn) {
+        closeModalBtn.click();
+      }
+
+      var showAddSuccess = document.getElementById('add-success-alert');
+      if(showAddSuccess){
+        showAddSuccess.style.display = "block";
+      }
+
+      setTimeout(function() {
+        if(showAddSuccess){
+          showAddSuccess.style.display = "none";
+        }
+      }, 4000);
```

Comment:
4000 -> wartości używane wielokrotnie w komponencie powinny być wyniesione do zmiennej typu **const** na początku wnętrza komponentu, w celu ułatwienia szybszej konfiguracji i zapewnienia, że wszystkie miejsca zaktualizujesz

W przypadku w którym w wielu komponentach pojawi się konieczność korzystania z takiej zmiennej powinien powstać jakiś oddzielny plik (np. `configuration.service.ts`), który będize zawierał w środku takie zmienne i będziesz mógł sobie zaimportować configuration i wywałać np. `GetNotificationTimeSpan() `który zwróci Ci te 4000 -> jakby co mogę pomóc to zrobić na następnym callu, póki co do **const'a**

Reply:
Dodano `const msTimeoutDelay = 4000;`